api/src/pcapi/models/feature.py

```diff
@@ -253,6 +253,21 @@ def install_feature_flags() -> None:
     db.session.commit()
 
 
+def clean_feature_flags() -> None:
+    """Automatically remove old feature flags from database.
+
+    This is done after each deployment and in tests.
+    """
+    installed_flag_names = {f[0] for f in Feature.query.with_entities(Feature.name).all()}
+    defined_flag_name = {f.name for f in list(FeatureToggle)}
+
+    to_remove_flags = installed_flag_names - defined_flag_name
+
+    for flag in to_remove_flags:
+        db.session.execute(text("DELETE FROM feature WHERE name = :name").bindparams(name=flag))
```

Comment:
Ne pourrait-on pas aggréger les requêtes de DELETE ? Un truc du genre :
 ```suggestion
    db.session.execute(text("DELETE FROM feature WHERE name = ANY(:names)").bindparams(names=to_remove_flags))
```

Reply:
Je n'arrive pas a faire marcher cette syntaxe pour plusieurs FF. De toute façon il sera tres rare que la boucle fasse plus d'un ou deux tours donc je ne pense pas qu'il soit pertinent de perdre trop de temps a faire cette opti.